librz/core/cmd_analysis.c

```diff
@@ -11,7 +11,7 @@ static const char *help_msg_a[] = {
 	"a*", "", "same as afl*;ah*;ax*",
 	"aa", "[?]", "analyze all (fcns + bbs) (aa0 to avoid sub renaming)",
 	"a8", " [hexpairs]", "analyze bytes",
-	"ab", "[b] [addr]", "analyze block at given address",
+	"ab", "[?] [b] [addr]", "analyze block at given address",
```

Comment:
no, we don't need the `b`. There is not only the `b`, but many others. So `[?]` is ok. With newshell it will be automatically printed by the code.